charts/aad-pod-identity/templates/identities.yaml

```diff
@@ -2,17 +2,29 @@
 apiVersion: "aadpodidentity.k8s.io/v1"
 kind: AzureIdentity
```

Comment:
Why there is AzureIdentity and AzureIdentityBinding as part of the chart?

Reply:
This is if the user wants to create an azure identity and binding while deploying the chart. By default this is false.

Reply:
What about making it optional? Create the identity and the binding only if requested?

Reply:
This parameter in values.yaml seems to be optional by default
`azureIdentity.enabled`         | Create azure identity and azure identity binding resource                                                                                                 | `false`

Reply:
https://github.com/Azure/aad-pod-identity/blob/f42188b065ea22dffed5df45e4f495fdc1703734/charts/aad-pod-identity/templates/identities.yaml#L1